src/menu/menu.scss

```diff
@@ -4,19 +4,23 @@
 @import '../inputs/variables';
 @import '../icon/index';
 
-@mixin iui-menu {
+@mixin iui-dropdown {
   @include iui-reset;
   @include iui-font-family;
   width: 100%;
```

Comment:
I think this is causing problems. It's taking extra width that it's not supposed to, which somehow makes it wider than the section itself.

![image](https://user-images.githubusercontent.com/9084735/116583368-45ae6900-a8e4-11eb-83da-c66804b5b393.png)

Reply:
Good catch!  I've moved `width: 100%;` from the dropdown mixin to the list-menu mixin.

Reply:
Don't forget to update your comment above so Vilius knows it's fixed 😄